TrackingTools/TrackAssociator/python/DetIdAssociatorESProducer_cff.py

```diff
@@ -33,7 +33,7 @@
     etaBinSize = cms.double(0.125),
     nEta = cms.int32(48),
     nPhi = cms.int32(48),
-    includeBadChambers = cms.bool(False)
+    includeBadChambers = cms.bool(True)
```

Comment:
a change made globally in this file will introduce problems in run1 simulation.
Run-2 specific changes should be added in postLS1 customizations